pkg/edgeview/src/tcp.go

```diff
@@ -75,6 +77,11 @@ const (
 	tcpMaxMappingNUM  int           = 5
 	tcpIdleTimeoutSec float64       = 1800.0
 	tcpCheckTimeout   time.Duration = 300 * time.Second
+	tcpKubeEndpoint   string        = "localhost:6443"
```

Comment:
Are these - localhost and port 6443 - defined elsewhere? Or are we assuming them? We must know them when we deploy the cluster, I assume there is a kubeconfig? Can we inherit them from somewhere rather than hard-coding?